homeassistant/components/growatt_server/sensor/total.py

```diff
@@ -48,5 +50,6 @@
         api_key="nominalPower",
         native_unit_of_measurement=UnitOfPower.WATT,
         device_class=SensorDeviceClass.POWER,
+        state_class=SensorStateClass.TOTAL,
```

Comment:
Same here, this is wrong. See https://github.com/home-assistant/core/blob/bb7dc079ce6160ebdd9828864ac9f8df255083b7/homeassistant/components/sensor/const.py#L637